server/prompt.go

```diff
@@ -11,8 +11,11 @@ import (
 	"github.com/ollama/ollama/template"
 )
 
+// chatPrompt accepts a list of messages and returns the prompt and images that should be used for the next chat turn.
```

Comment:
I remember these comments from another PR, I'm not sure if you meant to include them in this change.